component/client/backgroundContext.tsx

```diff
@@ -0,0 +1,4 @@
+const backgroundContext: any = chrome.extension.getBackgroundPage();
```

Comment:
This file groups all reference to background variables.
So it is easy to swap out in the future.